### PR TITLE
[ZH] Add `osu! music licensing`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by TicClick** [comment]
please only check the second item if there is a review from a native (or fluent) speaker, which is posted here, on GitHub

**Comment by FlyingFish23** [comment]
> please only check the second item if there is a review from a native (or fluent) speaker, which is posted here, on GitHub
Got it, thanks

**Comment by FlyingFish23** [comment]
> please only check the second item if there is a review from a native (or fluent) speaker, which is posted here, on GitHub

Got it, thanks.

**Review by heipizhu4 — COMMENTED** [review]
One point to discuss and an extra space to delete.

File: wiki/Legal/Music_licensing/zh.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+提交谱面是，谱面的制作者均会被推荐遵照我们的[内容使用准则](/wiki/Rules/Content_Usage_Guidelines)来向谱面中元素的相关著作权人取得使用许可。
+
+## 精选艺术家的版权政策
+
+osu! 取得使用许可的音乐内容均可从[精选艺术家列表](https://osu.ppy.sh/beatmaps/artists)当中以`.osz`（即谱面模板）的格式被免费下载。艺术家页面的每首曲目还会有自动生成的一分钟预览可供串流聆听。因为曲目仅作为社区用户的制谱选择，所以部分曲目不会被立刻用作游戏内容。
+
+我们期待且将会获得曲目在游戏内，相关网站和视频内容的相关使用权。这不包括在任何第三方可盈利平台的相关使用。部分著作权人可能会在那些情况下要求收益分成（在平台支持的情况下，例如 YouTube）。
+
+osu! 获得的全部音乐使用权均不是独家的，osu! 可使用的音乐，其他平台也可以在取得著作权人许可的情况下使用（我们始终相信，好音乐不应该被限制在 osu! 这一个平台里）。考虑到使用的性质，我们通常愿意购买歌曲的永久使用权（用户希望自己创作的谱面能够永久可用）。版税会在协商之后一次性付清。
```
Comment (YumeMuzi) [inline wiki/Legal/Music_licensing/zh.md:29]:
```suggestion
osu! 不会买断歌曲的使用权。这些歌曲既可以在 osu! 内使用，也可以在取得著作权人许可后，在其他平台内使用（我们始终相信，好音乐不应被限制在 osu! 这一个平台里）。考虑到使用的性质（玩家希望自己创作的谱面能够永久可用），我们通常会购买歌曲的永久使用权。版税会在协商之后一次性付清。
```

File: wiki/Legal/Music_licensing/zh.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## 联系我们
+
+**电子邮件:** [music@ppy.sh](mailto:music@ppy.sh)
+
+如果你想就你的内容在 osu! 内被使用所应取得的相关许可进行协商，请使用上面的电子邮件地址联系我们，并附带你的作品样览。我们会决定你的曲目适不适合作为 osu! 的内容，并且就版税问题与你达成统一。
```
Comment (YumeMuzi) [inline wiki/Legal/Music_licensing/zh.md:35]:
```suggestion
如果你对于加入 osu! 精选艺术家感兴趣，请使用上面的电子邮件地址联系我们，并附带你的作品样览。我们会决定你的曲目是否适合作为 osu! 的内容，并会与你洽谈版税相关事宜。
```

File: wiki/Legal/Music_licensing/zh.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+legal: true
+---
+
+# osu! 的音乐版权
+
+osu! 全部的内容（通常被称为"[谱面](/wiki/Beatmap)"或游戏关卡）都是由社区用户上传的。虽然我们会尽力指导用户获得作品的使用许可，但上传的音乐/画作仍然有一部分没有正确取得著作权人的许可。
+
+osu! 从不把盈利作为目标之一  ———— 我们非常努力不对上传的内容作宣传并且不从中获利（来避免通过用户的上传内容规避滥用版权作品应承担的法律责任），但我们知道，这往往不够。
+
+是这些艺术家们的作品才帮助 osu! 走到今天，而为了表示我们的支持，我们从用户手中收取到的捐赠资金结余均会投入到音乐版税当中。我们正在努力联系相关的著作权人们（包括以前内容和新内容的相关著作权人）来获取引用作品的版权且告知我们的[精选艺术家](/wiki/Featured_Artists)项目。
```
Comment (YumeMuzi) [inline wiki/Legal/Music_licensing/zh.md:11]:
```suggestion
是这些艺术家们的作品才帮助 osu! 走到今天，为了表示我们的支持，我们从玩家手中收取到的捐赠资金结余均会投入到音乐版税当中。我们正在努力联系相关的著作权人（包括以前内容和新内容的相关著作权人）来获取被使用内容的版权，并且向他们推荐我们的[精选艺术家](/wiki/Featured_Artists)项目。
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
Please fix these paragraphs for more fluent, precise, and comprehensible.

File: wiki/Legal/Music_licensing/zh.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+我们非常鼓励有意加入精选艺术家项目的艺术家们联系 osu! 团队来洽谈相关版权事宜。
+
+## osu! 对音乐的使用
+
+作为一款节奏游戏，osu! 用户上传的每一个关卡均会与一首歌的节奏相契合。
+
+用户们可以从[谱面列表](https://osu.ppy.sh/beatmapsets)页面搜索艺术家名称，作品名称或其他相关信息，并且下载有关谱面。多数谱面都会包含一首歌曲，用户创建的谱面数据，一张背景图片和游戏音效。
+
+提交谱面是，谱面的制作者均会被推荐遵照我们的[内容使用准则](/wiki/Rules/Content_Usage_Guidelines)来向谱面中元素的相关著作权人取得使用许可。
```
[the remark this answers is not in the document]
Comment (FlyingFish23) [reply wiki/Legal/Music_licensing/zh.md:21]: this article is for copyright holders/artists. some of them may not know what is "谱师", i think “谱面的制作者" would be more intuitive for them to understand.

File: wiki/Legal/Music_licensing/zh.md (hunk 1 above)
Comment (FlyingFish23) [reply wiki/Legal/Music_licensing/zh.md:29]: i don't think "买断" can't express the meaning of "独家". for example, i can "买断" a software and use it for a lifetime, but others can still purchase the same software. "买断" means one-time purchase, and “独家" means "only one can use". a better way for this sentence can be "osu! 不会购买歌曲的独家使用权".

Comment (YumeMuzi) [reply wiki/Legal/Music_licensing/zh.md:29]: ok

File: wiki/Legal/Music_licensing/zh.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/Legal/Music_licensing/zh.md:21]: ok

File: wiki/Legal/Music_licensing/zh.md (hunk 1 above)
Comment (YumeMuzi) [reply wiki/Legal/Music_licensing/zh.md:29]: Please update the rest of this suggested change

File: wiki/Legal/Music_licensing/zh.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/Legal/Music_licensing/zh.md:21]: Please update the rest of this suggested change, Most importantly, there is a typo in this sentence

**Comment by TicClick** [comment]
@YumeMuzi could you please give it another review and see if it reads well enough?

**Review by heipizhu4 — APPROVED** [review]
It's all done. Let's get it merged.

**Review by TicClick — APPROVED** [review]

**Pull request merged**